CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+## Release 0.2.0 - 2022/08/06
+
+#### Argument Changes:
+- Added argument `kraken_viral_id` with the default Kraken2 ID of `10239`
+
+#### Output Changes:
+- All samples CSV:
+    - Renamed the following columns:
+        - proportion_basecalled  -> genome_completeness
+        - total_n_count          -> num_consensus_n
+        - avg_read_depth         -> mean_sequencing_depth
+    - Added `median_sequencing_depth` column
+    - Added `percent_viral_reads` column for when Kraken2 is ran (not there otherwise)
+
+#### Other Changes:
+- Added changelog
+- Added testing reports for host removal metrics
+- Modified workflows
+    - Initial analysis workflow added (for fastqc)
+    - Host removal workflow emits kraken2 results if run
```

Comment:
omits

Reply:
I should re-word that slightly as it is a bit confusing the way I wrote it. It does emit a channel when ran though
```nextflow
    emit:
    kraken_results = ch_kraken_results      // channel: [ val(sampleID), path(krakenReport) ]
```